test/integration-test.py

```diff
@@ -195,6 +267,12 @@ def main():
     if not CONFIG_NEXT:
         run_expired_authz_purger()
 
+    # Run the boulder-janitor. This should happen after all other tests because
+    # it will advance the fake clock and delete rows that may otherwise be
```

Comment:
```suggestion
    # it runs with the fake clock set to the future and deletes rows that may otherwise be
```

(I found "advance the fake clock" a little confusing because there's no persistent fake clock between invocations in the integration test).

Reply:
Thanks for the suggested wording. Implemented in f4bf8f3